functions/utils/cache.test.ts

```diff
@@ -0,0 +1,37 @@
+import CacheMock from 'browser-cache-mock'
+
+//import { mocked } from '../../src/test-utils/mocked'
+import Cache from './cache'
+
+const cacheMock = new CacheMock()
+
+const data = {
+  title: 'test',
+  image: 'testImage',
+  url: 'testUrl',
+}
+
+beforeAll(() => {
+  const globalAny: any = global
+  globalAny.caches = {
+    open: async () => cacheMock,
+    ...cacheMock,
+  }
+})
+
+test('Should put cache properly', async () => {
+  jest.spyOn(cacheMock, 'put')
+  const response = await Cache.match('https://example.com')
+  expect(response).toBeUndefined()
+  await Cache.put(data, 'https://example.com')
+  expect(cacheMock.put).toHaveBeenCalledWith('https://example.com', expect.anything())
```

Comment:
check that it's being called with a response that matches data.
you can do this by getting the actual arg and then parsing it: cacheMock.put.calls...